### PR TITLE
Remove Denkensk from Kueue reviewers

### PR DESCRIPTION
**Pull request by PBundyra**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
As part of the effort to ensure new PRs are reviewed by the active reviewers, and trying to minimize notifications for the inactive ones. I contacted @denkensk and we aligned on the path forward

The change is based on [the repo activity dashboard](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-repo_name=kubernetes-sigs%2Fkueue&var-country_name=All) and follows k8s community guidelines

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by PBundyra** [comment]
Thank you so much for all of your contributions @denkensk!

File: OWNERS [hunk 1]
```diff
@@ -5,7 +5,6 @@ approvers:
 - mimowo
 
 reviewers:
-- denkensk
```
Comment (mimowo) [inline OWNERS:8]: I would suggest transferring to emeritus_reviewers to keep the recognition

**Comment by denkensk** [comment]
Thank you for this wonderful experience and everyone involved

File: OWNERS (hunk 1 above)
Comment (PBundyra) [reply OWNERS:8]: We already have `emeritus_approvers` section where denkensk is present. I believe it's even stronger recognition

Comment (mimowo) [reply OWNERS:8]: sure, I missed that, makes sense

**Comment by mimowo** [comment]
/lgtm
/approve
Thank you @denkensk for helping to start the project! Always feel free to contribute.

**Comment by tenzen-y** [comment]
> Thank you for this wonderful experience and everyone involved

@denkensk Thank you for all your efforts!

**Comment by tenzen-y** [comment]
@PBundyra Could you open the cherry-pick PRs against release-0.9 and release-0.8?

**Pull request merged**